.github/workflows/openvex.yaml

```diff
@@ -0,0 +1,35 @@
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        type: string
+        required: true
+        description: "GitHub release tag"
+
+permissions: {}
+
+jobs:
+  vexctl:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set env
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
```

Comment:
Hi @sreeram-venkitesh 
I am helping you to figure out what is different from this [kueue/pull/1475/files](https://github.com/kubernetes-sigs/kueue/pull/1475/files) PR, as a few of the CI checks are failing here.

Remove this `Set env` field.